### PR TITLE
Add a configuration for codeclimate

### PR DESCRIPTION
**Pull request by coolo**
Disable warnings about webui2 code and other places

### CODE REVIEW

**Comment by coolo** [comment]
There are 24 'new' issues, but in general the issue list is closer to what we'd want to fix.

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: "2"
+exclude_patterns:
+  - "dist/"
+  - "src/api/spec/"
```
Comment (hennevogel) [inline .codeclimate.yml]: Have you seen problems with this?

File: .codeclimate.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  - "src/api/test/"
+  - "docs/api/restility"
+  - "src/api/app/assets/javascripts/webui/application/cm2"
+  - "src/api/app/assets/javascripts/webui2"
```
Comment (hennevogel) [inline .codeclimate.yml]: Not sure this is the right way around. I think we should ignore `webui` instead.

File: .codeclimate.yml (hunk 1 above)
Comment (coolo) [reply .codeclimate.yml]: No, but this is part of the default exlude_patterns of codeclimate.

File: .codeclimate.yml (hunk 2 above)
Comment (coolo) [reply .codeclimate.yml]: Makes sense.

File: .codeclimate.yml (hunk 1 above)
Comment (hennevogel) [reply .codeclimate.yml]: I see

**Comment by coolo** [comment]
8 new - but they look reasonable. Please rereview

**Pull request merged**